src/js/index.js

```diff
@@ -0,0 +1,100 @@
+function preventFormDefault(menuForm){
+    menuForm.addEventListener('submit', (e) => {
+        e.preventDefault();
+    })
+}
+
+function countMenuItem($menuItem, menuCount){
+    menuCount.innerText = `총 ${$menuItem.length}개`
+}
+
+function addMenuItem(InputMenuName, menuList){
+    if(InputMenuName.value === '') {alert('메뉴를 입력해주세요.');}
```

Comment:
[제안]
if keyword 다음에 띄어쓰기를 해주면 좋을 것 같습니다.

```suggestion
    if (InputMenuName.value === '') { alert('메뉴를 입력해주세요.'); }
```

else 블록은 중괄호 다음 line break가 되었는데 이 부분은 안되어 있어서 통일성을 주면 좋을 것 같아요!

```suggestion
    if (InputMenuName.value === '') {
      alert('메뉴를 입력해주세요.');
    }
```

Reply:
예시로 적어주셔서 감사해요! 바로 이해했어요 :)